### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 - es. 3.21 (28 ottobre)

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/calcolo-delle-probabilita#304] [Esercizi M-Z]
<img width="596" alt="image" src="https://github.com/user-attachments/assets/b2eaecc8-730b-4f07-99a1-c70b32106267">

**Comment by alem1105** [comment]
<img width="450" alt="image" src="https://github.com/user-attachments/assets/58c601a9-53e3-4666-8ce1-3102d298a943">

**Reply by rimaout** [reply]
Anche a me stessi risultati

<img width="300" alt="image" src="https://github.com/user-attachments/assets/700c56cb-3371-44cf-9167-246a5f9b1785">
